app/models/podcast.rb

```diff
@@ -1,7 +1,9 @@
 require "text_sanitizer"
 
 class Podcast < ApplicationRecord
-  FEED_ATTRS = %i[subtitle description summary url new_feed_url display_episodes_count display_full_episodes_count enclosure_prefix enclosure_template feed_image itunes_image ready_feed_image ready_itunes_image ready_image]
+  FEED_ATTRS = %i[subtitle description summary url new_feed_url display_episodes_count
+    display_full_episodes_count enclosure_prefix enclosure_template feed_image itunes_image
+    ready_feed_image ready_itunes_image ready_image itunes_category itunes_subcategory itunes_categories]
```

Comment:
I couldn't figure out how to fixup this writeable collection in the Podcast representer:

```ruby
collection :itunes_categories, decorator: Api::ITunesCategoryRepresenter, class: ITunesCategory
```

So I ended up just making this one of the proxied-to-default-feed methods.

Reply:
not super knowledgeable about representers, but is this because you would need to move the collection to a `FeedRepresenter` but that doesn't exist? (what would a `FeedRepresenter` be used for, if anything? or is that even necessary?)

Reply:
It's tied up in the [representable gem](https://trailblazer.to/2.1/docs/representable) (which we're moving away from) and it's syntax.

Publish relies on the there being JSON that looks like `{"itunesCategories": [...]}` with that at the root of the JSON.  And I couldn't figure out any way to do some sort of `collection :itunes_categories, value: represented.default_feed.itunes_categories`.  It's non-trivial to pass in an explicit value I guess?  Representable seems to always want a property on the model called exactly what you're returning in the JSON.